pkg/kv/kvserver/rangefeed/catchup_scan.go

```diff
@@ -242,3 +269,37 @@ func (i *CatchUpIterator) CatchUpScan(
 	// Output events for the last key encountered.
 	return outputEvents()
 }
+
+// catchupScanRangeTombstones runs a catchup scan for range tombstones.
+func (i *CatchUpIterator) catchUpScanRangeTombstones(outputFn outputEventFn) error {
```

Comment:
What is the reason for not filtering rangekeys on timestamp here? Point values are only sent when they are newer than catchup timestamp. Maybe that deserves a comment.